src/common/config.h

```diff
@@ -17,9 +17,11 @@ class Config final {
 
     static Config From(const std::string &config_str);
 
-    std::vector<Endpoint> GetOtherEndpoints() const { return other_endpoints_; }
+    const std::map<int64_t, Endpoint>& GetOtherEndpoints() const { return other_endpoints_; }
```

Comment:
use unordered_map instead of map?

Reply:
Both fine，the general number of nodes is not too much